### ISSUE TITLE
[optimization] Move longitude and latitude data to a separate CSV

### ISSUE THREAD

**Issue by mpolidori** [datasets/covid-19#1]
As a user of the covid-19 data, I want the latitude and longitude data in a separate CSV file, so that it optimizes the use of the data by cutting down the file sizes, loading times, etc.

## Acceptance criteria

- [ ] Latitude and longitude data is moved to a separate CSV file
- [ ] A new datapackage.json is created for the new CSV
- [ ] A new visualization is created for it

**Comment by mok0** [comment]
I agree, it is redundant to include the lat, lon coordinates of locations in every line of the .csv file.
